src/app/model-poller.js

```diff
@@ -13,6 +17,7 @@ export default async function connectAndListModels(reduxStore, bakery) {
     reduxStore.dispatch(updateControllerConnection(conn));
     reduxStore.dispatch(updateJujuAPIInstance(juju));
     reduxStore.dispatch(updatePingerIntervalId(intervalId));
+    fetchControllerList(reduxStore);
```

Comment:
This may be a temporary location for this request.

We may want to gate this on access level so we don't have a silently failing request but we'll also want to use these values in the overall model list for users who have access at the controller level.